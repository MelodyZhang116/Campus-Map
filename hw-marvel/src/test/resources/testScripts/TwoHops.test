LoadGraph marvel marvel.csv

FindPath marvel INVISIBLE-WOMAN/SUE SABRA/RUTH-BAT-SERAP
FindPath marvel SPIDER-MAN/PETER-PAR MR.-FANTASTIC/REED-R