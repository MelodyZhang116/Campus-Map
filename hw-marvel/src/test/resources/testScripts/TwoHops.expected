loaded graph marvel

path from INVISIBLE-WOMAN/SUE to SABRA/RUTH-BAT-SERAP:
INVISIBLE-WOMAN/SUE to SABRA/RUTH-BAT-SERAP via COC-3
path from SPIDER-MAN/PETER-PAR to MR.-FANTASTIC/REED-R:
SPIDER-MAN/PETER-PAR to MR.-FANTASTIC/REED-R via Q-23